extensions/script-libraries/dropbox/dropbox.livecodescript

```diff
@@ -4318,10 +4318,11 @@ private command __API pHeader, pPost, pURL, pCallback
       
       if tError is empty then
          return tPostReturn for value
-      end if
-   end if
```

Comment:
@macMikey I was just going to poke you about the docs patches for this but I just noticed another issue. If tsNetPost returns an error it was previously being returned but with your patch it will not be. If you reinstate the  `end if`'s above and then just make the below as follows then it will do the right thing:

```
if tPostReturn is not empty then 
   put CR&tPostReturn after tError
end if
return tError for error
```

Reply:
Whoops - line out of position.  Fixed
What's missing on documentation?  It looks like my release note is in the PR.